test/e2e/README.md

```diff
@@ -5,18 +5,52 @@ libsinsp. Said tests are based around 2 containers:
 - One running the `sinsp-example` binary
 - Another one running the actual tests and verifying their outcome.
 
-## sinsp container
+## Running the tests
+The e2e tests use docker to run a few containers and execute commands in them
+making it relatively easy to ensure events are the one we are looking for by
+checking the `container.id` field in events. As the runner, pytest is used for
+its flexibility and ease of configuration and a Python module is used to hold
+common helper functions.
+
+### Install Python dependencies
+It is recommended to use a virtual environment for installing Python
+dependencies in order to prevent polluting your host. In order to create a
+virtual environment and install all dependencies in it, from the `test/e2e/`
+directory run the following commands:
+
+```sh
```

Comment:
To avoid accidentally not reading the explanation above and confusion in which directory you should be would suggest the following or similar:

```sh
cd /tmp;
git clone https://github.com/falcosecurity/libs.git;
pip3 install -r libs/test/e2e/tests/requirements.txt;
pip3 install libs/test/e2e/tests/commons;
```

Alternatively these steps could be included in the `make e2e-tests` which could be nicer and easier.

Reply:
I can add a `e2e-install-deps` target, but I wouldn't make it a dependency of `e2e-tests` for 2 reasons:
- The target would try to re-install dependencies every time you run `make e2e-tests`, it's not a big deal but does take additional time and it adds up.
- Someone could run the `e2e-tests` target and suddenly get stuff installed in their machine without their consent.